### PR TITLE
fix: Group MicrosoftAppId and MicrosoftPassword together in the settings page

### LINKED ISSUE #3046: Group MicrosoftAppId and MicrosoftPassword together in the settings page
## Describe the bug

The settings page looks like this right now, it's not convenient to update the MicrosoftAppId and MicrosoftPassword. We'd better group them together. 

{
  "feature": {
    "UseTranscriptLoggerMiddleware": false,
    "UseShowTypingMiddleware": false,
    "UseInspectionMiddleware": false,
    "UseCosmosDbPersistentStorage": false
  },
  "MicrosoftAppId": "",
  "luis": {
    "name": "",
    "authoringRegion": "westus",
    "defaultLanguage": "en-us",
    "environment": "composer",
    "authoringKey": "",
    "endpointKey": ""
  },
  "publishTargets": [],
  "qna": {
    "knowledgebaseid": "",
    "endpointkey": "",
    "hostname": ""
  },
  "telemetry": {
    "logPersonalInformation": false,
    "logActivities": true
  },
  "runtime": {
    "customRuntime": false,
    "path": "",
    "command": ""
  },
  "downsampling": {
    "maxImbalanceRatio": 10,
    "maxUtteranceAllowed": 15000
  },
  "MicrosoftAppPassword": ""
}

## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

### PR DESCRIPTION
**Pull request by VanyLaw**
## Description
Add schema to settings editor, make it more structured.

## Task Item

close #3046

### CODE REVIEW

File: Composer/packages/server/src/models/settings/defaultSettingManager.ts [hunk 1]
```diff
@@ -13,8 +13,108 @@ import { FileSettingManager } from './fileSettingManager';
...
+              type: 'number',
+            },
+            maxUtteranceAllowed: {
+              type: 'number',
+            },
+          },
+        },
+      },
+      default: this.createDefaultSettings(),
+    };
```
Comment (a-b-r-o-w-n) [inline Composer/packages/server/src/models/settings/defaultSettingManager.ts:117]: Can we define this in another file and import?

Comment (VanyLaw) [reply Composer/packages/server/src/models/settings/defaultSettingManager.ts:117]: done

**Comment by cwhitten** [comment]
@VanyLaw I think this does a bit more than it needs to. We shouldn't enforce a particular structure. What we want to do is when the default settings are generated, we take a best effort approach to put the id and password keys next to each other.

**Comment by VanyLaw** [comment]
@cwhitten I close it and reopen in https://github.com/microsoft/BotFramework-Composer/pull/3181

**Pull request closed without merging**